.changeset/silly-clocks-tan.md

```diff
@@ -15,9 +15,9 @@ To make the details of schema compatibilities that SharedTree supports more clea
 `TreeView.error` has been functionally replaced with the `compatibility` property.
 Users desiring the previous strict behavior should use `view.compatibility.isEquivalent` at appropriate places in application logic.
 
-# `ITree.schematize` deprecation
+# `ITree.schematize` removal
```

Comment:
I thought it made more sense to modify the existing changeset around migration than create a new one linking this one, given the perspective from someone using only released versions of FF is that this API will have just been immediately removed.